lib/sonos-http-api.js

```diff
@@ -67,7 +67,27 @@ function HttpAPI(discovery, settings) {
     }
 
     const params = req.url.substring(1).split('/');
-    
+
+    // Change node environment log level on request. 
+    if (/^\/loglevel\/.+$/.test(req.url)) {
```

Comment:
This could be simplified with `req.url.startsWith('/loglevel/')`, no need for a regexp.